<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TositelajiDialogi</class>
 <widget class="QDialog" name="TositelajiDialogi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>301</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tositelajin muokkaus</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Tunnus (lyhenne)</string>
       </property>
       <property name="buddy">
        <cstring>tunnusEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="tunnusEdit">
       <property name="maxLength">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Nimi</string>
       </property>
       <property name="buddy">
        <cstring>nimiEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="nimiEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Oletusvastatili</string>
       </property>
       <property name="buddy">
        <cstring>vastatiliEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="TilinvalintaLine" name="vastatiliEdit">
       <property name="placeholderText">
        <string>Kirjausten oletustasetili</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Oletustili</string>
       </property>
       <property name="buddy">
        <cstring>oletusTiliEdit</cstring>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="TilinvalintaLine" name="oletusTiliEdit">
       <property name="placeholderText">
        <string>Kirjausten oletustili (meno- tai tulotili)</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QGroupBox" name="lajiRyhma">
     <property name="title">
      <string>Tositteella kirjataan</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QRadioButton" name="kaikkiRadio">
        <property name="text">
         <string>Kaikentyyppisi√§ kirjauksia</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="ostoRadio">
        <property name="text">
         <string>Vain ostolaskuja (menotilit)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="myyntiRadio">
        <property name="text">
         <string>Vain myyntilaskuja (tulotilit)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="tilioteRadio">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Tiliotteita</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TilinvalintaLine</class>
   <extends>QLineEdit</extends>
   <header>db/tilinvalintaline.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tunnusEdit</tabstop>
  <tabstop>nimiEdit</tabstop>
  <tabstop>oletusTiliEdit</tabstop>
  <tabstop>vastatiliEdit</tabstop>
  <tabstop>kaikkiRadio</tabstop>
  <tabstop>ostoRadio</tabstop>
  <tabstop>myyntiRadio</tabstop>
  <tabstop>tilioteRadio</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TositelajiDialogi</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TositelajiDialogi</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
